docs/configure-custom-values-scalardb-analytics-postgresql.md

```diff
@@ -0,0 +1,179 @@
+# Configure a custom values file for ScalarDB Analytics with PostgreSQL
+
+This document explains how to create your custom values file for the ScalarDB Analytics with PostgreSQL chart. For details on the parameters, see the [README](https://github.com/scalar-labs/helm-charts/blob/main/charts/scalardb-analytics-postgresql/README.md) of the ScalarDB Analytics with PostgreSQL chart.
+
+## Required configurations
+
+### Database configurations
+
+You must set `scalardbAnalyticsPostgreSQL.databaseProperties`. Please set `database.properties` to this parameter. For more details on the configurations of ScalarDB, see [ScalarDB Configurations](https://github.com/scalar-labs/scalardb/blob/master/docs/configurations.md).
+
+```yaml
+scalardbAnalyticsPostgreSQL:
+  databaseProperties: |
+    scalar.db.contact_points=localhost
+    scalar.db.username=${env:SCALAR_DB_USERNAME:-}
+    scalar.db.password=${env:SCALAR_DB_PASSWORD:-}
+    scalar.db.storage=cassandra
+```
+
+### Namespaces configurations
```

Comment:
This might cause confusion between ScalarDB's namespace and Kubernetes's namespace.

Reply:
Ah, I see. I agree with that concern.
I will update the documents.
Thank you for pointing it out!

Reply:
I updated the document to describe the `namespace` is the `database namespace` in this context explicitly in 831fa610740cbced854e3409c22c4c8afd4a25d1.